templates/default/consul/ping-rdrs.sh.erb

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
+
+PROTO="http"
+echo ^d | openssl s_client -connect <%= node['ndb']['rdrs']['rest']['bind_ip'] %>:<%= node['ndb']['rdrs']['rest']['bind_port'] %>  > /dev/null 2>&1  && PROTO="https"
+test_command="curl -k -s --fail  $PROTO://<%= node['ndb']['rdrs']['rest']['bind_ip'] %>:<%= node['ndb']['rdrs']['rest']['bind_port'] %>/<%= node['ndb']['rdrs']['version'] %>/ping > /dev/null 2>&1"
```

Comment:
I saw that you are using the same thing in `rdrs-start.sh` script. What I would do is keep only the pinging stuff in this file - not the Consul retryable function.

Then create `rdrs-health` or something which would use the `_run_health_check`. 
That way you won't have to copy&paste the snippet to two files